deeplearning/train.py

```diff
@@ -0,0 +1,54 @@
+import tensorflow as tf
+import numpy as np
+import util.pixnet as pixnet
+
+train_size = 10
+image_size = [625, 1250, 3]
+vocab_size = 52
```

Comment:
Current vocab size is 49. I can give you links to all the data and stuff too since the `data/` folder is currently just being gitignored.